poseidon2-compress-air/src/air.rs

```diff
@@ -0,0 +1,211 @@
+use core::borrow::Borrow;
+
+use p3_air::{Air, AirBuilder, BaseAir};
+use p3_field::PrimeCharacteristicRing;
+use p3_matrix::Matrix;
+use p3_poseidon2::GenericPoseidon2LinearLayers;
+use p3_poseidon2_air::{Poseidon2Air, RoundConstants};
+
+use crate::{Poseidon2CompressCols, num_cols};
+
+/// Extends the Poseidon2 AIR with recursion circuit-specific columns and constraints.
```

Comment:
```suggestion
/// Extends the Poseidon2 AIR with recursion circuit-specific columns and lookups.
```
Because I can't see any change in the constraints.